docs/maui/views/RatingView.md

```diff
@@ -0,0 +1,519 @@
+| ShapeBorderColor | `Color` | Gets or sets the border color of the rating item shape. The default value of this is Grey.  This is a bindable property. |
+| ShapeBorderThickness | `Thickness` | Gets or sets the border thickness of the rating item shape.  The default value is a Thickness with all values set to 1.  This is a bindable property. |
+| ItemShapeSize | `double` | Gets or sets the size of the rating item shape.  The default value is 20. |
+| Shape | `RatingViewShape` | Gets or sets the rating item shape.  The property is of type [`RatingViewShape`](#set-shape) and is an enumeration. The default value is Star.  This is a bindable property. |
+| Spacing | `double` | Gets or sets the spacing between rating item elements.  The default value is 10.  This is a bindable property. |
+| Style | `Style` | Gets or sets the unique Style for this element. |
+| StyleClass | `IList<string>` | Gets or sets the style classes for the element. |
+| StyleId | `string` | Gets or sets a user defined value to uniquely identify the element. |
+| TranslationX | `double` | Gets or sets the X translation delta of the element. This is a bindable property. |
+| TranslationY | `double` | Gets or sets the Y translation delta of the element. This is a bindable property. |
+| Triggers | `IList<TriggerBase>` | Gets the list of TriggerBase objects associated to this element. This is a read-only bindable property. |
+| VerticalOptions | `LayoutOptions` | Gets or sets the LayoutOptions that define how the element gets laid out in a layout cycle. This is a bindable property. |
+| Visual | `IVisual` | Gets or sets a IVisual implementation that overrides the visual appearance of an element. This is a bindable property. |
+| Width | `double` | Gets the current width of this element. This is a read-only bindable property. |
+| WidthRequest | `double` | Gets or sets the desired width override of this element. This is a bindable property. |
+| Window | `Window` | Gets the Window that is associated with an element. This is a read-only bindable property. |
+| X | `double` | Gets the current X position of this element. This is a read-only bindable property. |
+| Y | `double` | Gets the current Y position of this element. This is a read-only bindable property. |
+| ZIndex | `int` | Gets or sets the front-to-back z-index of an element within a layout. This is a bindable property. |
+
+
+## Set custom shape
+The `CustomShape` property is a `string` that allows for the defining of custom rating item shape `path`. This feature empowers developers to implement unique designs, such as distinctive symbols, as rating items.
+
+Only when the `Shape` property is set to `Custom`, will the custom shape path be used.
+
+The following examples sets the custom and shape properties:
+
+```xaml
+<toolkit:RatingView
+	CustomShape="M 12 0C5.388 0 0 5.388 0 12s5.388 12 12 12 12-5.38 12-12c0-6.612-5.38-12-12-12z"
```

Comment:
What shape does this render? Do we have a screenshot of this?

Reply:
I didn't want to flood with loads of screen shots, as there is LOADS in the Samples app.  However, in this instance, you are right, and I'll get a screen shot.